src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/ProblemService.java

```diff
@@ -80,7 +80,7 @@ public Problem problemCreateToDb(CreateProblem cp) {
     }
 
     public List<ProblemResponse> getProblems() {
-        final List<Problem> problems = problemRepository.findAll();
+        final List<Problem> problems = problemRepository.getAllSortedBySeverityAndCreatedAt();
```

Comment:
we do not have paging there?

Reply:
If we would like to add pagination here it would require more effort, because our frontend is not adjusted to that yet. It's not in the scope of the story.